bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Tracker.java

```diff
@@ -142,6 +142,7 @@ public Tracker (Display display, int style) {
 	}
 	this.style = checkStyle (style);
 	this.display = display;
+	this.nativeZoom = display.getDeviceZoom();
```

Comment:
```suggestion
	this.nativeZoom = display.getNativeDeviceZoom();
```

Reply:
👍 Makes sense.

In that case:
* The `@since` tags could also be removed (since it's a protected method)
* `org.eclipse.swt.internal.DPIUtil.setDeviceZoom(int)` should also be renamed to `setNativeDeviceZoom`.

@akoch-yatta propose a separate PR for those changes please.

Reply:
> * The `@since` tags could also be removed (since it's a protected method)

Which method do you mean?

Reply:
I meant the `@since` in `org.eclipse.swt.widgets.Display.getDeviceZoom()`

Reply:
Adding protected methods requires a `@since` tag, so that one is correct. Maybe you are mixing up _package protected_ and _protected_ methods. Only the former don't need that tag as they are "internal".

Reply:
I'm confused. 
There is not getNativeDeviceZoom() in Display (or Device). getDeviceZoom() is the one that is returning the (native) zoom of the primary method.

Reply:
I am so sorry for introducing this confusion 🙁 I mixed up the methods in `Display` with those in `DPIUtil`...

My comments w.r.t. the `@since` are still valid, they properly refer to the `Display` class.